meshtastic/interfaces/ble/compatibility_service.py

```
@@ -364,7 +519,10 @@ def _publish_status() -> None:
     # COMPAT_STABLE_SHIM: retained for existing callers during service migration.
     @staticmethod
     def publish_connection_status_legacy(
-        iface: "BLEInterface", connected: bool, *, publishing_thread: object
+        iface: "BLEInterface",
+        connected: bool,
```

Ruff (0.15.6)

[warning] Boolean-typed positional argument in function definition

(FBT001)